src/client/componentsMealsharing/Reservations.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import "../App.css";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+function Reservations() {
+  const [fetchData, setFetchData] = useState([]);
```

Comment:
Maybe name this something different like 
```suggestion
  const [availableReservations, setAvailableReservations] = useState([]);
```